charts/sftp-server/values.yaml

```
@@ -117,6 +117,9 @@ service:
   port: 22
   # -- When using the NodePort type, you can specify a fixed port ([Kubernetes docs](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport))
   nodePort: 0 
+  externalTraffic:
+    enabled: true
```

I suggest to disable it by default, as that's also the k8s default.

```suggestion
    enabled: false
```

---

Fixed